teams/wg-parallel-rustc.toml

```diff
@@ -3,8 +3,9 @@ subteam-of = "compiler"
 kind = "working-group"
 
 [people]
-leads = ["Mark-Simulacrum", "nikomatsakis"]
-members = ["nikomatsakis", "Mark-Simulacrum", "alexcrichton", "cuviper", "spastorino"]
+leads = ["cjgillot"]
+members = ["cjgillot", "SparrowLii", "nnethercote", "bjorn3", "Kobzol"]
+alumni = ["nikomatsakis", "Mark-Simulacrum", "alexcrichton", "cuviper", "spastorino"]
```

Comment:
Nit: it might be nice to format these with each item on a new line. This makes diffs nicer.

Reply:
Thanks! It has been corrected.